### PR TITLE
Added support for Kerui PIR sensor

### PR DESCRIPTION
**Pull request by klattimer**
Based on Akhan remote keyless entry system

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 ########################################################################
 # Build utility
 ########################################################################
-add_executable(rtl_433 
+add_executable(rtl_433
 	baseband.c
```
Comment (merbanan) [inline src/CMakeLists.txt]: Stray change

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -93,7 +94,7 @@ target_link_libraries(rtl_433
     ${CMAKE_THREAD_LIBS_INIT}
 )
 
-	
+
 set(INSTALL_TARGETS rtl_433)
```
Comment (merbanan) [inline src/CMakeLists.txt]: Stray change.

File: src/devices/akhan_100F14.c [hunk 3]
```diff
@@ -55,6 +55,8 @@ static int akhan_rke_callback(bitbuffer_t *bitbuffer) {
 									"data",		"Data (4bit)",	DATA_FORMAT, 	"0x%x", 	DATA_INT, dataBits,
 									"other",		"Attention",	DATA_STRING,	"The data received is not used by the akham keyfob. This might be another device using a HS1527 OTP encoder with the same timing.",
 									NULL);
+			data_acquired_handler(data);
+			return 0;
 		}
```
Comment (merbanan) [inline src/devices/akhan_100F14.c]: Split this out to a separate patch.

File: src/devices/akhan_100F14.c [hunk 4]
```diff
@@ -79,7 +81,7 @@ PWM_Precise_Parameters pwm_precise_parameters_akhan = {
 
 r_device akhan_100F14 = {
 	.name          = "Akhan 100F14 remote keyless entry",
-	.modulation    = OOK_PULSE_PWM_PRECISE, 
+	.modulation    = OOK_PULSE_PWM_PRECISE,
 	.short_limit   = 316,
```
Comment (merbanan) [inline src/devices/akhan_100F14.c]: Stray change.

**Comment by merbanan** [comment]
Send a signal sample and change the comments and the patch is good to be merged.

**Comment by klattimer** [comment]
Ah shit, I wasn't finished! 

didn't expect you to merge it that fast, I've got another couple of reversals of stray changes and still need to capture a signal sample.

**Comment by merbanan** [comment]
You snooze you loose. The signal is still needed.

**Comment by klattimer** [comment]
heh, I've got a bunch more devices - one of which is an EBP 50 thermostat and boiler controller - it's apparently a 433Mhz device but doesn't show up (yet) need more tinkering time.

I was also wanting to help out as much as I can (couple hours a week at best) in making sure all the drivers have decent JSON encoders.

So expect more pull requests in future.

**Comment by merbanan** [comment]
Goodie, contributions are welcome.
